dbt_subprojects/dex/seeds/liquidity/uniswap_worldchain_base_liquidity_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_date,tx_hash,evt_index,block_number,id,token0,token1,amount0_raw,amount1_raw
+worldchain,uniswap,4,2025-02-24,0xd74c7edb67eacd58346060b743d7cc3501a5ba3e2a5cbe49fd2af379125743c6,1,10542625,0x43115bf1b6cb0cb257ff614842a798403625d8d35a52fb0063870db9a2815cd9,0x2cfc85d8e48f8eab294be644d9e25c3030863003,0x6335c1f2967a85e98ccc89da0c87e672715284db,24770637596337410,257705156941842480000
+worldchain,uniswap,4,2025-02-24,0xe3d08340fa157462316f3a662d2d5b97a7465d1320ba8db30c6df1edc3da32a8,1,10542480,0x43115bf1b6cb0cb257ff614842a798403625d8d35a52fb0063870db9a2815cd9,0x2cfc85d8e48f8eab294be644d9e25c3030863003,0x6335c1f2967a85e98ccc89da0c87e672715284db,69763886699102216,19398167902193880000
```

Comment:
Please check the seed:
https://github.com/duneanalytics/spellbook/actions/runs/13636742219/job/38117281428?pr=7766#step:13:83